src/main/java/roomescape/ReservationController.java

```diff
@@ -31,24 +34,34 @@ public String reservation(){
     @GetMapping("/reservations")
```

Comment:
여기 보면 /reservations이 대부분의 api에 반복되는데 이럴때는 controller 클래스에 @RequestMapping을 사용하면 반복을 줄일 수 있어요!
```suggestion
    @Controller
    @RequestMapping(value = "/reservations")
    public class ReservationController
    
    @GetMapping()
    @ResponseBody
    public ResponseEntity<List<Reservation>> reservations(){
    ...
    }

    @PostMapping()
     public ResponseEntity<Reservation> create(@RequestBody Reservation reservation) {
     ...
     }

    @DeleteMapping("/{id}")
    public ResponseEntity<Void> delete(@PathVariable Long id) {
    ...
    }
```

이런식으로요!